pom.xml

```diff
@@ -90,12 +89,6 @@
         <!-- Test Scope -->
         <dependency>
             <groupId>ch.qos.logback</groupId>
-            <artifactId>logback-core</artifactId>
-            <version>${logback.version}</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
             <version>${logback.version}</version>
```

Comment:
Since you dropped the logback-core, we don't need the ${logback.version} property anymore. Can you remove the logback.version property, and explicitly use the version here.